packages/resto-cohttp-client/resto-cohttp-client.1.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "contact@nomadic-labs.com"
+authors: [ "Nomadic Labs" "Ocamlpro" ]
+license: "MIT"
+homepage: "https://gitlab.com/nomadic-labs/resto"
+bug-reports: "https://gitlab.com/nomadic-labs/resto/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/resto"
+synopsis: "A minimal OCaml library for type-safe HTTP/JSON RPCs"
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" { >= "4.10" }
+  "dune" { >= "1.11" }
+  "uri" {>= "1.9.0" }
+  "resto-cohttp" {= version }
+  "cohttp-lwt" { >= "1.0.0" }
+  "lwt" { >= "3.0.0" & < "6.0.0" }
```

Comment:
Are you sure this will break with lwt 6.0? Breaking lwt releases have usually broken very few packages

Reply:
Hi @mseri, sorry for the delay of my response and thank you for having merge this PR!
Concerning the version of `lwt` package and checking the history of `resto` source tree, it seems `{ >= "3.0.0" & < "6.0.0" }` has been used for quite a long time without causing issues. Do you still think this should still be addressed anyway?
Thanks for your help!

Reply:
No that's fine, I was just curious. We can deal with it later if needed